cerbero/build/cookbook.py

```diff
@@ -411,12 +411,26 @@ def _recipe_status(self, recipe_name):
                     file_hash=recipe.get_checksum(), installed_files=[])
         return self.status[recipe_name]
 
+    def _recipe_is_reset(self, recipe_name):
+        '''
+        We need to use this method to check whether a recipe is reset
```

Comment:
This looks more like a comment than documentation for the method itself.
The documentation should explain what the method does, why you use might change over time :)